test/integ/deadline_job_attachments/test_job_attachments.py

```diff
@@ -1355,7 +1355,7 @@ def test_sync_inputs_bucket_wrong_account(
     )
 
     syncer = asset_sync.AssetSync(job_attachment_test.farm_id)
-    session_dir = tmp_path_factory.mktemp(r"An error occurred \(403\)")
```

Comment:
Was this written as a funky folder for a reason before?

Reply:
It doesn't make sense to have a funky folder since it doesn't have anything to do with what the test is testing

Reply:
Even if there is a reason, it's most likely not needed anymore as we already have a regex check to make sure char like "(" and "\" can't be used

Reply:
Perfect - this would make sense now.